src/mapcss/Condition.py

```diff
@@ -111,29 +111,31 @@ def get_sql(self):
             if params[0][:2] == "::":
                 return ("", "")
         try:
+            column_name = "tags->" + "'" + params[0] + "'"
```

Comment:
https://github.com/kothic/kothic/blob/b99e4ee9a7cd0c1fd014a651843d08aeb79e60c9/src/komap.py#L81 ?

Reply:
Made a change to use `escape_sql_column` in mvt_getter